src/urllib3/contrib/securetransport.py

```diff
@@ -124,7 +124,7 @@
 # Basically this is simple: for PROTOCOL_SSLv23 we turn it into a low of
 # TLSv1 and a high of TLSv1.2. For everything else, we pin to that version.
```

Comment:
I think the high is now TLSv1.3

Reply:
I'll update this. Also is `kTLSProtocol13` not available? I'm not sure when it was added to SecureTransport but I'm getting Illegal Parameter errors for using it. :(

Reply:
https://developer.apple.com/documentation/security/sslprotocol/ktlsprotocol13?language=objc mentions macOS 10.13+, which is what Travis uses. That's all I know, sorry!